Source/interfaces/IDolby.h

```diff
@@ -38,9 +38,44 @@ namespace Exchange {
                 DIGITAL_AC3 = 4,
                 AUTO = 5
             };
-            
+
+            enum SoundModes : uint8_t {
+                UNKNOWN,
+                MONO,
+                STEREO,
+                SURROUND,
+                PASSTHRU
+            };
+
+            /* @event */
+            struct EXTERNAL INotification : virtual public Core::IUnknown {
+                enum { ID = ID_DOLBY_OUTPUT_NOTIFICATION };
+
+                virtual ~INotification() {}
+                virtual void AudioModeChanged(const Dolby::IOutput::SoundModes, const bool) = 0;
+            };
+
+            virtual uint32_t Register(INotification*) = 0;
+            virtual uint32_t Unregister(INotification*) = 0;
+
+            // @property
+            // @brief Atmos capabilities of Sink
+            // @return supported: atmos supported or unsupported
+            virtual uint32_t AtmosMetadata(bool& supported /* @out */) const = 0;
+
+            // @property
+            // @brief Sound Mode - Mono/Stereo/Surround
+            // @return mode: sound mode
+            virtual uint32_t SoundMode(Dolby::IOutput::SoundModes& mode /* @out */) const = 0;
+
+            // @property
+            // @brief Enable Atmos Audio Output
+            // @param enable: enable/disable
+            virtual uint32_t EnableAtmosOutput(const bool& enable /* @in */);
```

Comment:
I guess you just forgot but it should be a pure abstract method (= 0)

"Dolby Digital Plus is a digital audio compression scheme. On the other hand, Dolby Atmos is the name of a surround sound technology." I don't think it is correct to treat DD+ and Atmos as the same thing. It is possible to have DD+ compression (aka E-AC-3) without Atmos MetaData . So, in that sense, Mode(DIGITAL_PLUS); Does not imply turning on Atmos Dolby.

Under the given assumption as stated above, I think it is fine to add this method since Mode(DIGITAL_PLUS) is not assumed to be "enabling" the Atmos.